packages/ERTP/src/issuerKit.js

```diff
@@ -14,25 +14,31 @@ import './types-ambient.js';
 
 /**
  * @template {AssetKind} K
+ * @typedef {object} IssuerRecord
+ * @property {string} name
+ * @property {K} assetKind
+ * @property {AdditionalDisplayInfo} displayInfo
+ * @property {Pattern} elementShape
+ */
+
+/**
+ * @template {AssetKind} K
+ * @param {IssuerRecord<K>} issuerRecord
  * @param {Baggage} issuerBaggage
- * @param {ShutdownWithFailure=} optShutdownWithFailure If this issuer fails
- * in the middle of an atomic action (which btw should never happen), it
+ * @param {ShutdownWithFailure | undefined} optShutdownWithFailure If this issuer
```

Comment:
```suggestion
 * @param {ShutdownWithFailure} [optShutdownWithFailure] If this issuer
```
is our preferred ts/jsdoc syntax for optional `@params`